examples/examples/compose_extrinsic_offline.rs

```diff
@@ -66,7 +66,7 @@ async fn main() {
 
 	// Send and watch extrinsic until in block (online).
 	let block_hash = api
-		.submit_and_watch_extrinsic_until(xt, XtStatus::InBlock)
+		.submit_and_watch_extrinsic_until_without_events(xt, XtStatus::InBlock)
```

Comment:
I'd use the new `submit_and_watch_extrinsic_until` here.

Reply:
Why? We don't check the report/events

Reply:
See https://github.com/scs/substrate-api-client/pull/606/files/6243e73241a007764a3f889237ee1a224f9e0cd3#r1253142071